infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -0,0 +1,21 @@
+const taskcluster = require('taskcluster-client');
+
+exports.tasks = [];
+exports.tasks.push({
+  title: 'Create role and expand smoketest',
+  requires: [],
+  provides: [
+    'role-create-expand',
+  ],
+  run: async () => {
+    let role = new taskcluster.Auth(taskcluster.fromEnvVars());
+    let randomId = taskcluster.slugid();
+    let roleId = `project:taskcluster:smoketest:${randomId}`;
+    let payload = {
+      description: 'smoketest test',
+      scopes: ['project:taskcluster:smoketest:*'],
+    };
+    await role.createRole(roleId, payload);
```

Comment:
Let's also check that the role expands correctly, by calling [auth.expandScopes](https://docs.taskcluster.net/docs/reference/platform/auth/api#expandScopes), passing scope `assume:${roleId}`.